### ISSUE TITLE
reproducing paper result

### ISSUE THREAD

**Issue by rinost081** [biplabpoudel25/CryoFSL#1]
Thank you for your work.

I have two questions regarding your work.

1. I have read your paper, and would like to reproduce same result. Is trained checkpoint (SAM2) available? 
2. Also, could you give us an instruction how you produced crYOLO and Topaz result?

Thank you!

**Comment by biplabpoudel25** [comment]
Please follow the steps below:

1. **Pretrained SAM2 Checkpoint**  
   Download the pretrained **SAM2 (hiera-large)** checkpoint from the official repository:  
   👉 [facebookresearch/sam2](https://github.com/facebookresearch/sam2)  
   We used this pretrained model directly for all CryoFSL experiments.

2. **crYOLO and Topaz Baselines**  
   The results for **crYOLO** and **Topaz** were obtained using the **standard setup and training configuration** from their respective official implementations.

**Comment by rinost081** [comment]
thank you for your answer!

i have one more question.  When you train/test in few shot setting, how did you split EMPIAR dataset. 

For example, when you pick 1 image for few shot training from 10028, did you pick the first index, or maybe randomly ...?

**Comment by rinost081** [comment]
I saw dataset preparation document, so I would like to make sure if my understanding is correct

#### 10028
- 1 shot setting
    - train: 061
    - val: -
    - test: 062 - 360

- 5 shot setting 
    - train: 061, 062, ..., 065
    - val: -
    - test: 066 - 360


thank you very much for your great work

**Comment by biplabpoudel25** [comment]
Hello. The split of the dataset into train, validation, and test set is random. Thanks.

**Comment by rinost081** [comment]
thank you! i could run train and test with your implementation.
